modules/apps/29-fee/types/keys.go

```diff
@@ -109,6 +109,22 @@ func ParseKeyFeesInEscrow(key string) (channeltypes.PacketId, error) {
 	return packetID, nil
 }
 
+// ParseKeyForwardRelayerAddress parses the key used to store the forward relayer address and returns the packetID
+func ParseKeyForwardRelayerAddress(key string) (channeltypes.PacketId, error) {
+	keySplit := strings.Split(key, "/")
+	if len(keySplit) != 4 {
+		return channeltypes.PacketId{}, sdkerrors.Wrapf(
+			sdkerrors.ErrLogic, "key provided is incorrect: the key split has incorrect length, expected %d, got %d", 4, len(keySplit),
+		)
+	}
+	seq, err := strconv.ParseUint(keySplit[3], 10, 64)
+	if err != nil {
+		return channeltypes.PacketId{}, err
+	}
+	packetID := channeltypes.NewPacketId(keySplit[2], keySplit[1], seq)
```

Comment:
nit: add spacing after closing curly braces

```suggestion
	}
	
	seq, err := strconv.ParseUint(keySplit[3], 10, 64)
	if err != nil {
		return channeltypes.PacketId{}, err
	}
	
	packetID := channeltypes.NewPacketId(keySplit[2], keySplit[1], seq)
```